.gitignore

```diff
@@ -5,8 +5,8 @@ mstpd
 
 /bridge-stp
 /bridge-stp.tmp
-/lib/ifupdown.sh
```

Comment:
alright ; let's remove the neutrality from this ; 
let's make it an ack from me (2 neutrals should convert to ack, in this case)